schema-next.yaml

```diff
@@ -16,6 +16,25 @@ versions:
               code.filepath: code.file.path
               code.lineno: code.line.number
               code.column: code.column.number
+        # https://github.com/open-telemetry/semantic-conventions/pull/TODO
+        - rename_attributes:
+            attribute_map:
+              db.system: db.system.name
+              db.cassandra.coordinator.dc: cassandra.coordinator.dc
+              db.cassandra.coordinator.id: cassandra.coordinator.id
+              db.cassandra.consistency_level: cassandra.consistency.level
+              db.cassandra.idempotence: cassandra.idempotence
+              db.cassandra.page_size: cassandra.page_size
+              db.cassandra.speculative_execution_count: cassandra.speculative_execution.count
+              db.cosmosdb.client_id: az.client.id
+              db.cosmosdb.connection_mode: az.cosmosdb.connection.mode
+              db.cosmosdb.consistency_level: az.cosmosdb.consistency.level
+              db.cosmosdb.request_charge: az.cosmosdb.request.charge
+              db.cosmosdb.request_content_length: az.cosmosdb.request.content_length
+              db.cosmosdb.regions_contacted: az.cosmosdb.request.regions_contacted
+              db.cosmosdb.sub_status_code: az.cosmosdb.response.sub_status_code
+              db.elasticsearch.node.name: elasticsearch.node.name
+              # db.elasticsearch.path_parts.<key> -> elasticsearch.path_parts.<key>
```

Comment:
is this commented out by mistake?

Reply:
commented out because it's a template attribute and rename transformation does not support them, added a comment about comment